src/react/client/components/FAB/index.tsx

```
@@ -1,10 +1,13 @@
 import { Settings } from 'lucide-react'
 import React, { useState } from 'react'
+import { toast } from 'sonner'
 
 import '@/styles/globals.css'
 import './styles.css'
 import { Drawers } from '@/client/components/drawers'
 import { ShortcutAssistantModal } from '@/client/components/shortcut-assistant-modal'
+import { Toaster } from '@/components/ui/sonner'
```

ESLint

[error] `@/components/ui/sonner` import should occur after import of `@/client/types/drawer`

(import/order)